packages/components/src/tree-select/stories/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * WordPress dependencies
+ */
+import { withState } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import TreeSelect from '../';
+
+export default { title: 'Components/TreeSelect', component: TreeSelect };
+
+const TreeSelectStory = withState( {
```

Comment:
Because of the way storybook renders the code in the Docs view when clicking "View code" it is a little more straight-forward if you only handle the state-specific things in the wrapper component. If you name the rest in the TreeSelectStory and pass them into the actual `TreeSelect` as `{ ...props } ` it makes it a little easier to understand there. 

<img width="1278" alt="Components___TreeSelect_-_Default_⋅_Storybook_and_Slack___general___GVSU_Web_Team_1" src="https://user-images.githubusercontent.com/6653970/75557093-6db6e980-5a0d-11ea-8a05-1229a95074ce.png">

Reply:
Looks great!